.github/workflows/npm-publish.yaml

```
@@ -0,0 +1,40 @@
+name: Publish to NPM
+
+on:
+  push:
+    tags:
+      - '*'
```

Shouldn't we expect tags in the SemVer standard here?

```suggestion
on:
  push:
    tags:
      - '*.*.*'
      - '*.*.*-staging'
```

---

yes! thanks for noticing!